user-guides/how-to-whitelist-a-token.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: 'How to Add a Token to the Whitelist'
+---
+
+To use a specific token as a reward for boosts on Boost Protocol, the token must be added to the approved whitelist. This process ensures the stability, security, and compliance of tokens used within the Boost ecosystem.
+
+## Whitelist Criteria
+
+To be considered for the whitelist, tokens need to meet certain criteria. Here are a few key factors we consider when evaluating tokens:
+
+- Token liquidity and market capitalization
+- Smart contract security and audit status
+- Regulatory compliance
+- Project stability and reputation
+
+## Application Process
+
+If you would like to have a token added to the reward token whitelist, please follow these steps:
+
+1. **Review Requirements**: Ensure the token meets the basic criteria mentioned above.
+
+2. **Submit Application**: Fill out the token whitelist application form [here](). You will need to provide:
```

Comment:
Lets also make sure we include a link here before merging this in!

May need to sync up with Matt on that

Reply:
yep, will make sure to get the form setup before merging